src/test/scala/org/broadinstitute/dsde/firecloud/service/LibraryServiceSpec.scala

```diff
@@ -315,25 +316,24 @@ class LibraryServiceSpec extends FreeSpec with LibraryServiceSupport with Attrib
     }
     "when validating JSON Schema" - {
       "fails on an empty JsObject" in {
-        val testSchema = FileUtils.readAllTextFromResource("test-attribute-definitions.json")
+        val testSchema = FileUtils.readAllTextFromResource("library/attribute-definitions.json")
         val sampleData = "{}"
         val ex = intercept[ValidationException] {
           validateJsonSchema(sampleData, testSchema)
         }
-        assertResult(13){ex.getViolationCount}
+        assertResult(23){ex.getViolationCount}
       }
       "fails with one missing key" in {
-        val testSchema = FileUtils.readAllTextFromResource("test-attribute-definitions.json")
+        val testSchema = FileUtils.readAllTextFromResource("library/attribute-definitions.json")
         val defaultData = testLibraryMetadata.parseJson.asJsObject
         val sampleData = defaultData.copy(defaultData.fields-"library:datasetName").compactPrint
         val ex = intercept[ValidationException] {
           validateJsonSchema(sampleData, testSchema)
         }
         assertResult(1){ex.getViolationCount}
-        assert(ex.getMessage.contains("library:datasetName"))
```

Comment:
This assert doesn't work any more.  Because of the more complicated validation, these always fail with something like "only 1 subschema matches out of 2".